homeassistant/components/climate/sensibo.py

```diff
@@ -0,0 +1,235 @@
+            'temperatureUnit'] == 'C' else TEMP_FAHRENHEIT
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        return {ATTR_CURRENT_HUMIDITY: self.current_humidity}
+
+    @property
+    def temperature_unit(self):
+        """Return the unit of measurement which this thermostat uses."""
+        return self._temperature_unit
+
+    @property
+    def available(self):
+        """Return True if entity is available."""
+        return self._status
+
+    @property
+    def target_temperature(self):
+        """Return the temperature we try to reach."""
+        return self._ac_states['targetTemperature']
+
+    @property
+    def target_temperature_step(self):
+        """Return the supported step of target temperature."""
+        return 1
+
+    @property
+    def current_operation(self):
+        """Return current operation ie. heat, cool, idle."""
+        return self._ac_states['mode']
+
+    @property
+    def current_humidity(self):
+        """Return the current humidity."""
+        return self._measurements['humidity']
+
+    @property
+    def current_temperature(self):
+        """Return the current temperature."""
+        # This field is not affected by temperature_unit.
+        # It is always in C / nativeTemperatureUnit
+        if 'nativeTemperatureUnit' not in self._ac_states:
+            return self._measurements['temperature']
+        return convert_temperature(
+            self._measurements['temperature'],
+            TEMP_CELSIUS,
+            self.temperature_unit)
+
+    @property
+    def operation_list(self):
+        """List of available operation modes."""
+        return self._operations
+
+    @property
+    def current_fan_mode(self):
+        """Return the fan setting."""
+        return self._ac_states['fanLevel']
+
+    @property
+    def fan_list(self):
+        """List of available fan modes."""
+        return self._current_capabilities['fanLevels']
+
+    @property
+    def current_swing_mode(self):
+        """Return the fan setting."""
+        return self._ac_states['swing']
+
+    @property
+    def swing_list(self):
+        """List of available swing modes."""
+        return self._current_capabilities['swing']
+
+    @property
+    def name(self):
+        """Return the name of the entity."""
+        return self._name
+
+    @property
+    def is_aux_heat_on(self):
+        """Return true if AC is on."""
+        return self._ac_states['on']
+
+    @property
+    def min_temp(self):
+        """Return the minimum temperature."""
+        return self._current_capabilities['temperatures'][
+            'C' if self.unit_of_measurement == TEMP_CELSIUS else 'F'][
+                'values'][0]
+
+    @property
+    def max_temp(self):
+        """Return the maximum temperature."""
+        return self._current_capabilities['temperatures'][
+            'C' if self.unit_of_measurement == TEMP_CELSIUS else 'F'][
```

Comment:
Actually the JSON looks like this
```json
"C": {"isNative": True, "values": [16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30]},
"F": {"isNative": False, "values": [61, 63, 64, 66, 68, 70, 72, 73, 75, 77, 79, 81, 82, 84, 86]}}
```
So not all whole-F values are accepted.
I'll add a fix to make sure correct F temperature is set.

As for min and max - it would be safest to stick to the provided list. What is the advantage of converting?